express_relay/sdk/js/src/types.ts

```diff
@@ -89,14 +86,34 @@ export type Opportunity = {
    */
   targetCallValue: bigint;
   /**
-   * Tokens required to repay the debt
+   * Tokens required to execute the opportunity
    */
   sellTokens: TokenAmount[];
   /**
    * Tokens to receive after the opportunity is executed
    */
   buyTokens: TokenAmount[];
 };
+
+export type OpportunitySvm = {
+  order: OrderStateAndAddress;
+  program: "limo";
+  /**
+   * The chain id where the opportunity will be executed.
+   */
+  chainId: ChainId;
+  slot: number;
+  blockHash: Blockhash;
+};
+
+export type OpportunityParams = OpportunityEvm | OpportunitySvm;
+
+export type Opportunity = OpportunityParams & {
```

Comment:
Why not using the opportunity type exposed by the server?
I think it's not a good idea to redefine these types and we can remove some of these new types.

Reply:
The main difference here is having native js/ts types instead of strings/numbers for all fields.